### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pritchyspritch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+`$ brew install pre-commit` (on Mac)
+
+or
+
+`$ pip install pre-commit` (on Linux or Mac)
+
+### Configuring pre-commit
+We recommend using some of the plugins that come with the framework, as well as installing the [detect-secrets plugin](https://github.com/Yelp/detect-secrets).
+
+An example [.pre-commit-config.yaml file](https://github.com/alphagov/gds-pre-commit/blob/add-yaml/.pre-commit-config.yaml) used to set up your pre-commit config can be found in this repository.
```
Comment (alice-carr) [inline README.md]: we should tell people here to put this file in the root of their repo?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+Run the following command to install your pre-commit hooks to git:
+
+`$ pre-commit install`
+
+Run an initial check to ensure everything is installed and working correctly:
+
+`$ pre-commit run --all-files`
+
+Once that's completed, you're all done! If you want to add any other hooks, take a look at the [extensive list](https://pre-commit.com/hooks.html). There are various handy linters and formatters for Go, Ruby, Python and Terraform.
```
Comment (alice-carr) [inline README.md:48]: To update your hooks, periodically run:

`$ pre-commit autoupdate`

or something like that?

**Review by alice-carr — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,53 @@
...
+This repository is here to assist GDS users in setting up pre-commit hooks that can improve the quality and security of projects hosted on GitHub.
+
+## pre-commit framework
+One of the easiest ways to get started with pre-commit hooks is by using the [pre-commit framework](https://pre-commit.com/).
+
+### Installing pre-commit
+
+To get started you can either run:
+
+`$ brew install pre-commit` (on Mac)
```
Comment (issyl0) [inline README.md:11]: Homebrew on Linux works for installing `pre-commit`, too. 😉

Comment (pritchyspritch) [reply README.md:11]: Noted, you learn something new everyday. I missed this before I merged so I'm removing it from my next commit.

Comment (issyl0) [reply README.md:11]: ❤️

**Pull request merged**